Makefile

```diff
@@ -28,6 +29,12 @@ helm-lint: helm-lint-$(notdir $(CHART_DIR)/*)
 helm-lint-%:
 	$(HELM) lint $(CHART_DIR)/$*
 
+deploy-cert-manager:
+	kubectl create namespace cert-manager
+	$(HELM) install --wait cert-manager --namespace cert-manager jetstack/cert-manager --version v1.0.1 \
```

Comment:
Another way to install this would be as an optional dependency of helm-operator. What do you prefer @drewwells?

Reply:
This is guaranteed to fail on every run after the initial install. I'd keep it out of our chart and it needs to check if cert manager is already in the cluster. If you ran this on env-4, it's very likely to break certs in the cluster. In authz we check if we're in minikube before doing crd installs

Reply:
https://github.com/Infoblox-CTO/ngp.authz/blob/7ca636f56270a77fe4566336bf77deb278611469/Makefile

Reply:
My understanding is that the `deploy-cert-manager` target will only be used for testing in this repo. Since cert-manager already exists on env-4, and other envs by extension, we don't need to call this target at all. Please correct me if I am wrong.

Reply:
Yeah that's fine. I commented from my phone and thought it was being referenced in a more generic deploy rule.